installer/fix_log4j_cve.xml

```diff
@@ -0,0 +1,47 @@
+<project default="fix" name="log4j 1.2.X fix for CVEs">
+	<taskdef resource="org/apache/ant/compress/antlib.xml" classpath="${plugin_classpath}"/>
+	
+	<property name="log4j.version" value="${log4j.version}" />
+	<property name="log4j.path" value="${basedir}/target/repository/plugins/org.apache.log4j_${log4j.version}.jar" />
+	<property name="log4j.path.new" value="${java.io.tmpdir}/log4j_${log4j.version}.new.jar" />
+	<property name="artifacts.path.new" value="${java.io.tmpdir}/artifacts.new.jar/" />
+	<property name="artifacts.jar.path" value="${basedir}/target/repository/artifacts.jar" />
+	<property name="artifacts.xz.path" value="${basedir}/target/repository/artifacts.xml.xz" />
+
+	<target name="fix">
+		<echo>Removing files from Log4j bundle archive...</echo>
+		<jar destfile="${log4j.path.new}" manifest="log4jManifest.MF">
+			<zipfileset src="${log4j.path}" excludes="org/apache/log4j/net/JMSSink.class, org/apache/log4j/jdbc/JDBCAppender.class, org/apache/log4j/chainsaw/*, META-INF/*" />
+		</jar>
+		<delete file="${log4j.path}" />
+		<move file="${log4j.path.new}" tofile="${log4j.path}" />
+		
+		<echo>Removing Signature from repository...</echo>
```

Comment:
I would expect there is an Eclipse application (https://wiki.eclipse.org/Equinox/p2/Publisher#Features_And_Bundles_Publisher_Application) to do it in one step but if this work

Reply:
correct but needs an eclipse instance to republish everything. this is ugly but faster:)